src/manifolds/StiefelEuclideanMetric.jl

```diff
@@ -1,4 +1,8 @@
 
+default_inverse_retraction_method(::Stiefel) = PolarInverseRetraction()
```

Comment:
We should maybe document these, so they show up in the docs as well (just in case someone wonders what the default on Stiefel is)?

Reply:
Sure, done.